test/Conversion/FIRRTLToHW/intrinsics.mlir

```diff
@@ -176,4 +176,19 @@ firrtl.circuit "Intrinsics" {
     firrtl.strictconnect %hbr1, %0 : !firrtl.uint<1>
     firrtl.strictconnect %hbr2, %1 : !firrtl.uint<1>
   }
+
+  // CHECK-LABEL: hw.module @FPGAProbe
+  firrtl.module @FPGAProbe(
+    in %clock: !firrtl.clock,
+    in %reset: !firrtl.uint<1>,
+    out %out: !firrtl.uint<8>,
+    in %in: !firrtl.uint<8>
+   ) attributes {convention = #firrtl<convention scalarized>} {
+    %0 = firrtl.wire : !firrtl.uint<8>
+    %1 = firrtl.wire : !firrtl.clock
+    firrtl.int.fpga_probe %1, %0 : !firrtl.uint<8>
```

Comment:
You should assert here that the wire is present.

Reply:
Agree, this needs to check something. 😉